src/current/cockroachcloud/billing-management.md

```diff
@@ -61,4 +61,6 @@ Under the **Spend over time** section, you can see a table of each of your organ
 
 ## View invoices
 
-You can view all of your organization's past invoices on the **Invoices** tab of the **Billing** page. Download any invoice to see the details of your charges for a billing period.
+You can view all of your organization's past invoices on the **Invoices** tab of the **Billing** page. Click on any invoice to view the details of your charges for a billing period. 
+
+If you're using [folders]({% link cockroachcloud/folders.md %}) to organize your clusters, billing is grouped by folder. Click any folder to view invoices for individual clusters within it.
```

Comment:
```suggestion
If you're using [folders]({% link cockroachcloud/folders.md %}) to organize your clusters, billing is grouped by folder. Click any folder to view invoices for individual clusters in the folder or its descendants. Within a folder, click any subfolder to view its details.
```
Is this accurate? I think it's key that the costs roll up the hierarchy to the organization root. WDYT @jramos-crl

Reply:
That's accurate and yes makes it a bit clearer that it rolls up the hierarchy.